spec/conjurctl/commands/configuration/show_spec.rb

```diff
@@ -8,16 +8,31 @@ def command_output(config, format:)
     output_stream = StringIO.new
 
     Commands::Configuration::Show.new(
-      conjur_config: config,
       output_stream: output_stream
     ).call(
+      conjur_config: config,
       output_format: format
     )
 
     output_stream.string
   end
 
-  let(:conjur_config) { Conjur::ConjurConfig.new }
+  let(:conjur_config) { double(Conjur::ConjurConfig) }
+
+  let(:source_trace) {
+    {
+      trusted_proxies: {
+        source: { type: :defaults },
+        value: []
+      }
+    }
+  }
+
+  before do
+    allow(conjur_config).
+      to receive(:to_source_trace).
```

Comment:
What does this method do?  The name wasn't clear to me... is it part of AnywayConfig?

Reply:
Yep it's from AnywayConfig. It returns a hash with info about where each attribute is being read from.